PR #3657 broke macOS compilation when using '--enable-debug' flag

```
libtool: link: /Users/rex/zcash/depends/x86_64-apple-darwin17.7.0/share/../native/bin/ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -mmacosx-version-min=10.8 -stdlib=libc++ -std=c++11 -Wformat -Wformat-security -Wstack-protector -fstack-protector-all -fPIE -pipe -O1 -g -g3 -O0 -fwrapv -fno-strict-aliasing -pthread -Wl,-pie -Wl,-headerpad_max_install_names -Wl,-dead_strip -o test/test_bitcoin test/test_bitcoin-arith_uint256_tests.o test/test_bitcoin-addrman_tests.o test/test_bitcoin-alert_tests.o test/test_bitcoin-allocator_tests.o test/test_bitcoin-base32_tests.o test/test_bitcoin-base58_tests.o test/test_bitcoin-base64_tests.o test/test_bitcoin-bech32_tests.o test/test_bitcoin-bip32_tests.o test/test_bitcoin-bloom_tests.o test/test_bitcoin-checkblock_tests.o test/test_bitcoin-Checkpoints_tests.o test/test_bitcoin-coins_tests.o test/test_bitcoin-compress_tests.o test/test_bitcoin-convertbits_tests.o test/test_bitcoin-crypto_tests.o test/test_bitcoin-DoS_tests.o test/test_bitcoin-equihash_tests.o test/test_bitcoin-getarg_tests.o test/test_bitcoin-hash_tests.o test/test_bitcoin-key_tests.o test/test_bitcoin-dbwrapper_tests.o test/test_bitcoin-main_tests.o test/test_bitcoin-mempool_tests.o test/test_bitcoin-miner_tests.o test/test_bitcoin-mruset_tests.o test/test_bitcoin-multisig_tests.o test/test_bitcoin-netbase_tests.o test/test_bitcoin-pmt_tests.o test/test_bitcoin-policyestimator_tests.o test/test_bitcoin-pow_tests.o test/test_bitcoin-prevector_tests.o test/test_bitcoin-raii_event_tests.o test/test_bitcoin-reverselock_tests.o test/test_bitcoin-rpc_tests.o test/test_bitcoin-sanity_tests.o test/test_bitcoin-scheduler_tests.o test/test_bitcoin-script_P2SH_tests.o test/test_bitcoin-script_tests.o test/test_bitcoin-scriptnum_tests.o test/test_bitcoin-serialize_tests.o test/test_bitcoin-sighash_tests.o test/test_bitcoin-sigopcount_tests.o test/test_bitcoin-skiplist_tests.o test/test_bitcoin-test_bitcoin.o test/test_bitcoin-timedata_tests.o test/test_bitcoin-torcontrol_tests.o test/test_bitcoin-transaction_tests.o test/test_bitcoin-uint256_tests.o test/test_bitcoin-univalue_tests.o test/test_bitcoin-util_tests.o test/test_bitcoin-sha256compress_tests.o test/test_bitcoin-accounting_tests.o wallet/test/test_test_bitcoin-wallet_tests.o test/test_bitcoin-rpc_wallet_tests.o -Wl,-bind_at_load  -L/Users/rex/zcash/depends/x86_64-apple-darwin17.7.0/share/../lib libbitcoin_wallet.a libbitcoin_server.a libbitcoin_cli.a libbitcoin_common.a libbitcoin_util.a crypto/libbitcoin_crypto.a univalue/.libs/libunivalue.a -L/opt/local/lib -L/opt/local/lib/db48 ./leveldb/libleveldb.a ./leveldb/libmemenv.a -lboost_filesystem -lboost_program_options -lboost_thread -lboost_chrono -lboost_unit_test_framework secp256k1/.libs/libsecp256k1.a -L/Users/rex/zcash/depends/x86_64-apple-darwin17.7.0/lib /Users/rex/zcash/depends/x86_64-apple-darwin17.7.0/lib/libevent_pthreads.a /Users/rex/zcash/depends/x86_64-apple-darwin17.7.0/lib/libevent.a -ldb_cxx-6.2 -lssl libzcash.a snark/libsnark.a /Users/rex/zcash/depends/x86_64-apple-darwin17.7.0/lib/libgmpxx.a /Users/rex/zcash/depends/x86_64-apple-darwin17.7.0/lib/libgmp.a -lboost_system -lcrypto /Users/rex/zcash/depends/x86_64-apple-darwin17.7.0/lib/libsodium.a -lrustzcash -ldl /Users/rex/zcash/depends/x86_64-apple-darwin17.7.0/lib/libzmq.a -lstdc++ -lpthread -pthread
Undefined symbols for architecture x86_64:
  "FakeCoinsViewDB::DB_COINS", referenced from:
      FakeCoinsViewDB::GetCoins(uint256 const&, CCoins&) const in libbitcoin_wallet.a(libbitcoin_wallet_a-zcbenchmarks.o)
      FakeCoinsViewDB::HaveCoins(uint256 const&) const in libbitcoin_wallet.a(libbitcoin_wallet_a-zcbenchmarks.o)
Undefined symbols for architecture x86_64:
  "FakeCoinsViewDB::DB_COINS", referenced from:
      FakeCoinsViewDB::GetCoins(uint256 const&, CCoins&) const in libbitcoin_wallet.a(libbitcoin_wallet_a-zcbenchmarks.o)
      FakeCoinsViewDB::HaveCoins(uint256 const&) const in libbitcoin_wallet.a(libbitcoin_wallet_a-zcbenchmarks.o)
ld: symbol(s) not found for architecture x86_64
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [zcashd] Error 1
make[2]: *** Waiting for unfinished jobs....
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [test/test_bitcoin] Error 1
```

--- a/src/zcbenchmarks.cpp
+++ b/src/zcbenchmarks.cpp
@@ -453,9 +453,11 @@ double benchmark_increment_sapling_note_witnesses(size_t nTxs)
 // This class is based on the class CCoinsViewDB, but with limited functionality.
 // The construtor and the functions `GetCoins` and `HaveCoins` come directly from
 // CCoinsViewDB, but the rest are either mocks and/or don't really do anything.
+
+// The following constant is a duplicate of the one found in txdb.cpp
+static const char DB_COINS = 'c';
+
 class FakeCoinsViewDB : public CCoinsView {
-    // The following constant is a duplicate of the one found in txdb.cpp
-    static const char DB_COINS = 'c';
 
     CDBWrapper db;
 

